docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -90,6 +90,68 @@ Once the initial enrollment has completed, and if Teleport Identity
 is enabled, the enrollment workflow will guide you through enabling SCIM
 provisioning as well.
 
+## Configure Access List Synchronization
+
+If Teleport Identity is enabled, the enrollment workflow will also guide you to configure syncing of
+User Groups and Direct Application Assignments as access lists. This will ensure
+that permissions in Okta are modeled properly in Teleport, allowing Teleport users
+to see the same Okta applications that they would when viewing the Okta dashboard.
+
+### Configuring Default Owners
+
+![Set Default Owners](../../../img/enterprise/plugins/okta/okta-access-list-default-owners.png)
+
+Access lists must have owners, so the first step of setting up Okta Access List
+Synchronization is setting default owners for our new access lists. You can select
+any number of default owners and free-form enter the owners if they do not yet exist
+in the system. These owners can be later changed and will not be overwritten by the
+Okta Access List synchronization process.
+
+### Importing Okta User Groups as Access Lists
+
+![Import Okta User Groups](../../../img/enterprise/plugins/okta/okta-access-list-import-user-groups-all.png)
+
+After configurating default owners, you can customize which Okta User Groups should
+be imported. By default, all user groups will be imported. If you would like more
+control, you can untoggle the "Import All User Groups" toggler at the top right of
+the panel and configure multiple filters which will be matched against the group
+names.  For more information on filter syntax, refer to the Import Filters section
+below.
+
+<Admonition type="note">
+Only Okta User Groups with assignments will be imported as an Access List. If an Okta User
+Group has no assignments, it will not be imported until it has assignments.
+</Admonition>
+
+### Importing Okta Application Assignments as Access Lists
+
+![Import Okta Application Assignments](../../../img/enterprise/plugins/okta/okta-access-list-import-applications.png)
+
+In addition to importing Okta User Groups, you can also import direct application
+assignments within Okta as Access Lists as well. This behaves in exactly the same way
```

Comment:
Here you did the reverse of above. Access Lists is now capitalized but application assignments is not.

Reply:
@zmb3 should all mention of `access lists` be capitalized to `Access Lists`? I think i remember all product name should be capitalized, but I wasn't sure (we seem to cap `access lists`  most of the time in this PR)

Reply:
The intent was to capitalize all mentions of access list, so I'll go over this with a fine toothed comb and make sure they're all capitalized properly.

Reply:
Is that correct though? Access list is not a product name. It's just a noun like user or role, isn't it?

Reply:
IIRC the prior art is to use Access List instead of access list.